src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -21,7 +21,10 @@
         "NETStandard.Library": {
           "version": "1.6.1",
           "exclude": "runtime"
-        }
+        },
+        "Microsoft.NETCore.Platforms": "1.1.0",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0",
+        "Microsoft.NETCore.ILAsm": "2.0.0-beta-25027-03",
```

Comment:
It does cause this to be dumped out during init-tools:
```
Restoring BuildTools version 1.0.27-prerelease-01327-01...
Initializing BuildTools...
D:\git\corefx\packages\Microsoft.DotNet.BuildTools\1.0.27-prerelease-01327-01\lib\tool-runtime\project.json(26,52): warn
ing NU1012: Dependency conflict. Microsoft.NETCore.ILAsm 2.0.0-beta-25027-03 expected Microsoft.NETCore.Runtime.CoreCLR
>= 2.0.0-beta-25027-03 but received 1.1.0
```
which I don't really like that noise either so we really should look at other options. However for the auto-update build definition I'm fine with making that not an error.